addons/mail/security/ir.model.access.csv

```diff
@@ -66,3 +66,4 @@ access_res_users_settings_volumes_user,res.users.settings.volumes,model_res_user
 access_mail_template_reset,access.mail.template.reset,model_mail_template_reset,mail.group_mail_template_editor,1,1,1,1
 ir_actions_report_access_user,ir.actions.report.access.user,base.model_ir_actions_report,base.group_user,1,0,0,0
 access_mail_link_preview_admin,mail.link.preview.admin,model_mail_link_preview,base.group_erp_manager,1,1,1,1
+access_mail_gif_favorite,mail.gif.favorite,model_mail_gif_favorite,base.group_user,1,1,1,1

```

Comment:
There should be a corresponding ir rule to restrict access to only own favorites.